ui/public/images/hashicorp.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 106.86 113.14"><title>iAsset 1</title><g id="Layer_2" data-name="Layer 2"><g id="Logo"><polygon points="44.54 0 0 25.69 0 25.71 0 87.41 16.73 97.07 16.73 35.35 44.54 19.3 44.54 0"/><polygon points="62.32 0 62.32 49.15 44.54 49.15 44.54 30.81 27.8 40.47 27.8 103.44 44.54 113.12 44.54 64.11 62.32 64.11 62.32 82.33 79.05 72.67 79.05 9.66 62.32 0"/><polygon points="62.32 113.14 106.86 87.45 106.86 87.43 106.86 25.73 90.12 16.07 90.12 77.79 62.32 93.84 62.32 113.14"/></g></g></svg>
```

Comment:
We can tidy this markup a bit and add a `currentColor` property to simplify our CSS.

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 106.86 113.14" fill="currentColor"><polygon points="44.54 0 0 25.69 0 25.71 0 87.41 16.73 97.07 16.73 35.35 44.54 19.3 44.54 0"/><polygon points="62.32 0 62.32 49.15 44.54 49.15 44.54 30.81 27.8 40.47 27.8 103.44 44.54 113.12 44.54 64.11 62.32 64.11 62.32 82.33 79.05 72.67 79.05 9.66 62.32 0"/><polygon points="62.32 113.14 106.86 87.45 106.86 87.43 106.86 25.73 90.12 16.07 90.12 77.79 62.32 93.84 62.32 113.14"/></svg>
```